crates/ubrn_cli/src/ios.rs

```diff
@@ -213,8 +213,8 @@ impl IOsArgs {
             .arg(manifest_path)
             .arg("--target")
             .arg(&target.triple);
-        if self.common_args.release {
-            cmd.arg("--release");
+        if self.common_args.profile() != "debug" {
+            cmd.arg("--profile").arg(self.common_args.profile());
```

Comment:
Nit: extract a local variable
```suggestion
        let profile = self.common_args.profile();
        if profile != "debug" {
            cmd.args(["--profile", profile]);
```